diagnostic_feedback/tests/unit/base_test.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import unittest
+
+from webob import Request
+from workbench.runtime import WorkbenchRuntime
+from xblock.fields import ScopeIds
+from xblock.runtime import KvsFieldData, DictKeyValueStore
+
+import diagnostic_feedback
+
+
+
+# Silence too verbose Django logging
+logging.disable(logging.DEBUG)
```

Comment:
Don't change global config at import time. You can put this in `run_tests.py` or something like that.